dev-client/src/components/sheets/privacy/DataPrivacyContent.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+
+import {TranslatedParagraph} from 'terraso-mobile-client/components/content/typography/TranslatedParagraph';
+import {ExternalLink} from 'terraso-mobile-client/components/links/ExternalLink';
+import InternalLink from 'terraso-mobile-client/components/links/InternalLink';
+import {Column} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+export const DataPrivacyContent = () => {
+  const {t} = useTranslation();
+
+  return (
+    <Column space="12px">
```

Comment:
question: can/should we use `<View>` instead of `<Column>`?

Reply:
`Column` is used very frequently in the app and I would prefer to continue doing that. If we stop using it, I feel that the replacement should be some sort of standard layout component developed with designer input rather than an ad-hoc `View` element.

Reply:
Yeah I feel like the general approach we're planning with most of the design stuff is "keep following existing conventions until we have decided what we should do (aka what the component should be), and then start doing that for new stuff, and then update old stuff"